mods/d2k/maps/atreides-05/atreides05-AI.lua

```diff
@@ -0,0 +1,160 @@
+IdlingUnits = { }
+
+AttackGroupSize =
+{
+	easy = 6,
+	normal = 8,
+	hard = 10
+}
+
+AttackDelays =
+{
+	easy = { DateTime.Seconds(4), DateTime.Seconds(7) },
+	normal = { DateTime.Seconds(2), DateTime.Seconds(5) },
+	hard = { DateTime.Seconds(1), DateTime.Seconds(3) }
+}
+
+HarkonnenVehicleTypes = { "trike", "trike", "trike", "quad", "quad" }
+HarkonnenHeavyVehicleTypes = { "combat_tank_h", "missile_tank" }
```

Comment:
AI doesn't have an IX Lab in this mission. So it shouldn't be able to build Missile Tank.